ci-usd/README.md

```diff
@@ -56,6 +57,7 @@ Contains:
 * pyside-5.12.6
 * cmake-3.18.4
 * imath-2.4.0
+* materialx-
```

Comment:
Looks like something may be broken here and it didn't pull ASWF_MATERIALX_VERSION?  Or is it because this version of USD didn't require MaterialX?